src/utils/episodeSearchBoxFunc.ts

```diff
@@ -0,0 +1,17 @@
+import { IEpisode } from "../types";
+
+export function episodeSearchBoxFunc(
```

Comment:
This function name could mention that this function filters. It feels redundant to put Func in your function name as it is clear from where it is used that it is a function